compiler/rustc_session/src/parse.rs

```diff
@@ -79,7 +81,7 @@ impl SymbolGallery {
 /// The `feature`'s `Symbol` is the one you used in `unstable.rs` and `rustc_span::symbols`.
 #[track_caller]
 pub fn feature_err(
-    sess: &ParseSess,
+    sess: &Session,
```

Comment:
This is `parse.rs`, which is supposed to be about the `ParseSess`, so this isn't exactly accurate anymore. But I would prefer not moving those functions in this PR. Can you add a to-do comment about moving those functions out of this file and then do that in a follow-up PR?

Reply:
I've added a comment